lib/deps-for.js

```diff
@@ -8,13 +8,15 @@ var pkg = require('./pkg');
  * @method depsFor
  * @param {String} name of package to assemble unique deps for
  * @param {String} dir (optional) path to begin resolving from
+ * @param {Object} options (optional)
+ * @param {Boolean} options.includeOptionalDeps (optional) should optionalDependencies be included
  * @return {Array} a unique set of all deps
  */
-module.exports = function depsFor(name, dir) {
+module.exports = function depsFor(name, dir, options = {}) {
```

Comment:
This library still supports old versions of node which do not support default args. A future release will drop support, but for now we still support those old versions.

Mind fixing?

Reply:
I saw the changes in master, and have copied them into my version.